openedx.yaml

```diff
@@ -0,0 +1,8 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: uitk
+oeps: {}
+openedx-release: {ref: master}
+owner: edx/edx-ux-team
```

Comment:
You could make an @edx/fedx alias that points to me, Alasdair, Alisan, Andy, Dennis and Tyler